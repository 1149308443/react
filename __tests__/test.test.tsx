import NoFound from '@src/views/404/';
import { mount } from 'enzyme';

test("get child component AddTodo length", () => {
  // 模拟 props
  //   const props = {};

  // 通过 enzyme 提供的 shallow(浅渲染) 创建组件
  const Wrapper = mount(<NoFound />);
  expect(Wrapper.html()).toMatchSnapshot()
})

export { }