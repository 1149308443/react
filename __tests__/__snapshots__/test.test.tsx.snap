// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get child component AddTodo length 1`] = `"<div>111</div>"`;
